src/main/java/edu/umich/ctools/analytics/Course.java

```diff
@@ -0,0 +1,72 @@
+package edu.umich.ctools.analytics;
+
+import java.util.HashMap;
+
+public class Course {
```

Comment:
I like having these data classes.  It could be useful to attach some other methods to classes if those methods only act on information in this class.  E.g. make a csv line from the data for a particular course.